### PR TITLE
chore(Packit): Update Packit configuration

### PR DESCRIPTION
**Pull request by lbarcziova**
In case of any questions, please contact [Packit team](https://packit.dev/#contact).

For details, please see the commit messages.

### CODE REVIEW

**Comment by Tiboris** [comment]
Thanks for the patch @lbarcziova, I would like to kindly ask you to prepend the commit message with `chore`: `chore(Packit): $subj`

File: .packit.yaml [hunk 1]
```diff
@@ -42,6 +42,13 @@ actions:
     - "python3 setup.py --version"
 
 jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+    - fedora-all
+    - epel-8
+    - epel-9
+
```
Comment (Tiboris) [inline .packit.yaml:51]: And if possible change the targets to the format: 
```suggestion
    targets:
    - targets: [fedora-all, epel-8, epel-9]
```
so our lint does not cry at lout

**Review by Tiboris — CHANGES_REQUESTED** [review]
Oh and also some nitpick from our pre-commit hooks:

File: .packit.yaml (hunk 1 above)
Comment (Tiboris) [reply .packit.yaml:51]: If you do this manually, installing `pre-commit` to the repository helps to identify all the issues before a push.

Comment (lbarcziova) [reply .packit.yaml:51]: sure, will update it :) (it was done via a script)

**Review by Tiboris — APPROVED** [review]
Thanks for the updates, LGBTI

**Pull request merged**